doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1325 @@
+
+* Other:
+
+  * Google Find My Device that allows the creation of a locator device that is compatible with the Android `Find My Device app`_.
+    This feature is showcased in the :ref:`fast_pair_locator_tag` sample.
+  * Hardware model v2 (HWMv2), an improved extensible system for defining boards. This is the default boards definition system from this |NCS| release and onwards.
+    TBD add a link to the migration guide.
+  * :ref:`zephyr:sysbuild`, an improved and extensible system for multi-image build, replacing :ref:`ug_multi_image` (parent/child images).
+    See `Migrating from multi-image builds to sysbuild`_.
+  * Samples and applications that use short-range radio and run on multi-core SoCs were migrated to use the :ref:`ipc_radio` as the default image for the network/radio core.
+
+Added the following features as experimental:
+
+* Support for |NCS| v2.7.0 in nRF Connect for VS Code is experimental.
+  VS Code users that need v2.7.0 should use the preview version.
+  Please refer to VS Code for details.
+
+* nRF54H and nRF54L Series:
+
+  * Experimental support for next-generation devices.
+    Hardware access is restricted to the participants in the limited sampling program.
+    For more information, `contact our sales`_.
+
+* Bluetooth® LE:
+
+  * Path loss monitoring in the SoftDevice controller, which is part of LE Power Control.
+    This feature does not yet have a host stack API, it requires access via HCI commands.
+
+* Wi-Fi:
+
+  * Random MAC address generation at boot time in the nRF Wi-Fi driver.
+
+* DECT NR+:
+
+  * DECT NR+ PHY API which is showcased in the :ref:`nrf_modem_dect_phy_hello` sample.
+    To get access to the DECT NR+ PHY modem firmware, please `contact our sales`_.
+
+* nRF Cloud:
+
+  * The nRF Cloud CoAP library now uses the SO_KEEPOPEN socket option available on modem firmware 2.0.1 and above.
+    This option keeps the UDP socket open during temporary network outages, avoiding unnecessary handshakes and saving power and data.
+
+Sign up for the `nRF Connect SDK v2.7.0 webinar`_ to learn more about the new features.
+
+See :ref:`ncs_release_notes_270_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.7.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.7.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.0.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
```

Comment:
```suggestion
`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.0.
See the :ref:`installation` section for more information about supported operating systems and toolchain.

.. note::
     Because of the significant, potentially breaking changes introduced in the |NCS| v2.7.0, the support for v2.7.0 in the |nRFVSC| is experimental in the following areas:
     
     * sysbuild - During the extension stabilization period, manually edit and verify Kconfig option values instead of using the nRF Kconfig GUI.
     * Hardware Model v2 - During the extension stabilization period, edit devicetree files manually instead of using the Devicetree Visual Editor. Moreover, custom boards added in the extension will follow the Zephyr's Hardware Model v1.
     
     The extension users that need v2.7.0 should `switch to the pre-release version of the extension`_.

```